ui/src/app/views/account-creation/component.tsx

```diff
@@ -0,0 +1,294 @@
+                       onChange={e => this.updateProfile('familyName', e.target.value)}/>
+            {familyName.length > 80 &&
+            <ErrorMessage id='familyNameError'>
+              Last Name must be 80 character or less.
+            </ErrorMessage>}
+          </FormSection>
+          <FormSection>
+            <LongInput type='text' id='contactEmail' name='contactEmail'
+                       placeholder='Email Address'
+                       onChange={e => this.updateProfile('contactEmail', e.target.value)}/>
+          </FormSection>
+          <FormSection>
+            <LongInput type='text' id='currentPosition' name='currentPosition'
+                       placeholder='You Current Position'
+                       value={currentPosition}
+                       style={(currentPosition.length > 255) ?
+                         inputStyles.unsuccessfulInput : inputStyles.successfulInput}
+                       onChange={e => this.updateProfile('currentPosition', e.target.value)}/>
+            {currentPosition.length > 255 &&
+            <ErrorMessage id='currentPositionError'>
+              Current Position must be 255 characters or less.
+            </ErrorMessage>}
+          </FormSection>
+          <FormSection>
+            <LongInput type='text' id='organization' name='organization'
+                       placeholder='Your Organization'
+                       value={organization}
+                       style={(organization.length > 255) ?
+                         inputStyles.unsuccessfulInput : inputStyles.successfulInput}
+                       onChange={e => this.updateProfile('organization', e.target.value)}/>
+            {organization.length > 255 &&
+            <ErrorMessage id='organizationError'>
+              Organization must be 255 characters of less.
+            </ErrorMessage>}
+          </FormSection>
+          <FormSection style={{display: 'flex'}}>
+              <textarea style={{
+                ...inputStyles.formInput,
+                ...inputStyles.longInput,
+                'height': '10em',
+                'resize': 'none',
+                'width': '16rem'
+              }}
+                        id='areaOfResearch'
+                        name='areaOfResearch'
+                        placeholder='Describe Your Current Research'
+                        onChange={e => this.updateProfile('areaOfResearch', e.target.value)}/>
+            <TooltipTrigger content='You are required to describe your current research in
+                      order to help All of Us improve the Researcher Workbench.'>
+              <InfoIcon style={{
+                'height': '22px',
+                'marginTop': '2.2rem',
+                'paddingLeft': '2px'
+              }}/>
+            </TooltipTrigger>
+          </FormSection>
+          <FormSection>
+            <LongInput type='text' id='username' name='username' placeholder='New Username'
```

Comment:
The username input focus handling seems to have been dropped. SHould either be readded or we should have a ticket tracking it.

Reply:
Was this addressed or filed?